### ISSUE TITLE
Option to allow mount/dismount volume will be nice

### ISSUE THREAD

**Issue by ronaldsuwandi** [marta-file-manager/marta-issues#397]
Can be useful when mounting network share as well - currently the only way to do this is to mount it from Finder and then use Marta to access them

**Comment by VKondakoff** [comment]
I think this is a very important feature (together with the ability to use plain F8 to delete files on network shares).

**Comment by jt196** [comment]
Agreed - currently switching between this and Forklift due to lack of FTP and network share functionality.

**Comment by vurtomatic** [comment]
Would love this too, will replace Qspace Pro right away.
